src/Markdig.Tests/TestNormalize.cs

```diff
@@ -409,6 +409,13 @@ public void TaskLists()
             AssertNormalizeNoTrim("[ ] This is not a task list");
         }
 
+        [Test]
+        public void PipeTables()
+        {
+            AssertNormalizeNoTrim("| Foo | Bar |\n| :--- | ---: |\n| Hello World | *World* |");
```

Comment:
All of these asserts fail because the ColumnDefinition is not parsed correctly.

@xoofx Any ideas? I tried to understand the parsing logic but failed miserable ;)

Input: "| Foo | Bar |\n| :--- | ---: |\n| Hello World | *World* |"
Normalized Output: "| Foo | Bar |\n| :--- | ---: | --- |\n| Hello World | *World* |\n"